package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "webpack": "webpack",
+    "build": " ",
```

Comment:
@kleva-j  what's this please?

Reply:
It was a fix for an error preventing me from installing packages as there was no build script.